### PR TITLE
Add description of docker memory setting in node operator manual

### PR DESCRIPTION
**Pull request by martinetlee**
## Motivation and Context

This PR addresses https://github.com/quantstamp/qsp-protocol-node/issues/480 and would mention the docker memory setting in the manual.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: doc/node-operator.md [hunk 1]
```diff
@@ -143,6 +143,9 @@ the account to MetaMask so that you can stake QSP.
    
    Verify: `docker -v` should return Docker version 17.09.0-ce or above
 
+1. Check the memory setting of Docker (Docker Icon -> Preferences -> Advanced), it is recommended to set it above 16GB
```
Comment (alex-murashkin) [inline doc/node-operator.md]: `above 16 GB` -> `to 16 GB`. I am not sure if setting it higher is a good idea: the police node is using 16 GB.

Comment (martinetlee) [reply doc/node-operator.md]: Fixed! would need another approval. Thanks!

File: doc/node-operator.md [hunk 2]
```diff
@@ -143,7 +143,7 @@ the account to MetaMask so that you can stake QSP.
    
    Verify: `docker -v` should return Docker version 17.09.0-ce or above
 
-1. Check the memory setting of Docker (Docker Icon -> Preferences -> Advanced), it is recommended to set it above 16GB
+1. Check the memory setting of Docker (Docker Icon -> Preferences -> Advanced), it is recommended to set it to 16GB.
```
Comment (alex-murashkin) [inline doc/node-operator.md]: one more thing: need to mention it's on a MacOS. Is there an equivalent command on Linux? If so, let's add it as well.

Comment (martinetlee) [reply doc/node-operator.md]: Right, let me check if there's Linux command for doing this

Comment (martinetlee) [reply doc/node-operator.md]: According to the docker documentation: https://docs.docker.com/config/containers/resource_constraints/

"A container has no resource constraints and can use as much of a given resource as the host’s kernel scheduler allows."

Therefore there should be no limit on Linux environment.

**Pull request merged**